src/popover/popover.jsx

```diff
@@ -162,12 +162,10 @@ const Popover = React.createClass({
         if (nextProps.animated) {
           this.setState({closing: true});
           this._timeout = setTimeout(() => {
```

Comment:
Nah, your code is always clean. I never had a hard time reviewing your code :+1: